mod/account/set/right/password.rb

```diff
@@ -5,7 +5,7 @@
 PASSWORD_REGEX = {
   lower: /[a-z]/,
   upper: /[A-Z]/,
-  special_char: /[!\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~]/,
```

Comment:
@cschweipert I'm a little confused about how this passed before, but the main branch is busted. That must mean I approved and merged a branch that broke things. 

This seems to fix things; just wanted you to know.

Reply:
@ethn Oh. Thanks for fixing it.